app/assets/javascripts/discourse/lib/bootstrap-json/index.js

```diff
@@ -101,6 +101,26 @@ function beforeScriptLoad(buffer, bootstrap) {
   );
 }
 
+function discoursePreloadStylesheets(buffer, bootstrap) {
+  (bootstrap.stylesheets || []).forEach((s) => {
+    let attrs = [];
+    if (s.media) {
+      attrs.push(`media="${s.media}"`);
+    }
+    if (s.target) {
+      attrs.push(`data-target="${s.target}"`);
+    }
+    if (s.theme_id) {
+      attrs.push(`data-theme-id="${s.theme_id}"`);
+    }
+    if (s.class) {
+      attrs.push(`class="${s.class}"`);
+    }
```

Comment:
You're not actually using `attrs` anywhere 😛

Reply:
Yeah, you're right, sorry :laughing: removed.